pkg/ai/prompts.go

```diff
@@ -1,5 +1,9 @@
 package ai
 
 const (
-	default_prompt = "Simplify the following Kubernetes error message and provide a solution in %s: %s"
+	default_prompt = `Simplify the following Kubernetes error message delimited by triple dashes written in --- %s --- language; --- %s ---.
+	Provide the most possible solution in a step by step style in no more than 280 characters. Write the output in the following format:
+	Error:
```

Comment:
Is it really necessary to recall the error?

Reply:
Specifying this pattern for the output aims to have an explanation of the error. Happy to remove this if there's no value or we want to go with the idea of multiple prompts with different styles I proposed in Slack (for example non-technical, non-infra knowledge, etc.).